app/src/main/java/com/android/partagix/model/InventoryViewModel.kt

```diff
@@ -95,10 +98,17 @@ class InventoryViewModel(
             }
             val lenderUsersIds = mutableListOf<String>()
             val itemsBor = mutableListOf<Item>()
+            val availabilityBor = mutableListOf<Boolean>()
             loansList
                 .filter { it.idBorrower.equals(currentUser.uid) }
                 .filter { it.state == LoanState.ACCEPTED || it.state == LoanState.ONGOING }
                 .forEach { loan ->
+                  availabilityBor.add(
+                      if (loan.startDate.before(Date()) && loan.endDate.after(Date())) {
+                        true
+                      } else {
+                        false
+                      })
```

Comment:
For better readability: 

```suggestion
                  availability.add(loan.startDate.before(Date()) && loan.endDate.after(Date()))
```